src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,23 @@
 import React from 'react';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
-);
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => {
+  return (
+    <div className="movies">
+      {
+        movies.map(
+          (elementOfMovies: Movie) => (
+            <MovieCard key={elementOfMovies.imdbId} movie={elementOfMovies} />
+          ),
+        )
+      }
```

Comment:
Pay attention to your code formatting and naming. Please fix it like I mentioned before

Reply:
Okay. Thanks